src/functions/example-type.ts

```diff
@@ -0,0 +1,37 @@
+import * as AJV from 'ajv';
```

Comment:
I don’t understand what this file is, could you give it a quick explanation?

Reply:
Sure. @philsturgeon, this file is used by `valid-type-example` rule: https://github.com/stoplightio/spectral/blob/06fcb9e4cadc78b5c4299abd28d8f327f913f0f1/src/rulesets/oas3/ruleset.json#L2432
This rule takes care of the cases where we have:
```
type: some_type
example: some example
```
So where the shape of an example is defined by `.type`, not by `schema`.